lib/ansible/module_utils/basic.py

```diff
@@ -397,7 +422,7 @@ def __init__(self, argument_spec, bypass_checks=False, no_log=False,
         self.aliases = {}
 
         if add_file_common_args:
-            for k, v in FILE_COMMON_ARGUMENTS.iteritems():
+            for k, v in iteritems(FILE_COMMON_ARGUMENTS):
                 if k not in self.argument_spec:
```

Comment:
For all of these parameter iteritems calls it's probably better to just turn them into ```dict.items()```.  So that's this call and the rest of the calls in this file.